e2e/cypress/integration/integrations/oauth/oauth_spec.js

```diff
@@ -0,0 +1,440 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+import {checkboxesTitleToIdMap} from '../../enterprise/system_console/channel_moderation/constants';
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+import {
+    enablePermission,
+    goToSystemScheme,
+    saveConfigForScheme,
+} from '../../enterprise/system_console/channel_moderation/helpers';
+
+describe('Integrations page', () => {
+    let user1;
+    let user2;
+    let testChannelUrl1;
+    let oauthClientID;
+    let oauthClientSecret;
+
+    before(() => {
+        cy.requireWebhookServer();
+
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableOAuthServiceProvider: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            testChannelUrl1 = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+        });
+
+        goToSystemScheme();
+        enablePermission(checkboxesTitleToIdMap.ALL_USERS_MANAGE_OAUTH_APPLICATIONS);
```

Comment:
Since this is changing permission scheme, please add license check at the start of `before` hook:
```
cy.apiRequireLicense();
```

Also, please move file to `e2e/cypress/integration/enterprise` folder since we're organizing all enterprise tests in the said folder.

Reply:
Oopss, this has not resolve yet, but please move as I already approved the PR.  Thanks!

Reply:
Sorry, I forgot about the second part of the comment :sweat_smile: Will do right now.

Reply:
Done :smile: